src/emulator/gui/src/settings_dialog.cpp

```diff
@@ -48,13 +49,83 @@ bool change_pref_location(const std::string &input_path, const std::string &curr
     return true;
 }
 
+std::vector<std::string>::iterator modules;
+vfs::FileBuffer buffer;
+static ImGuiTextFilter search_bar;
+
 void draw_settings_dialog(HostState &host) {
     ImGui::PushStyleColor(ImGuiCol_Text, GUI_COLOR_TEXT_MENUBAR_OPTIONS);
     ImGui::Begin("Settings", &host.gui.configuration_menu.settings_dialog, ImGuiWindowFlags_AlwaysAutoResize);
     ImGuiTabBarFlags settings_tab_flags = ImGuiTabBarFlags_None;
     ImGui::BeginTabBar("SettingsTabBar", settings_tab_flags);
 
+    // Core
+    if (ImGui::BeginTabItem("Core")) {
+        ImGui::PopStyleColor();
+        if (vfs::read_file(VitaIoDevice::VS0, buffer, host.pref_path, "sys/external/libc.suprx")) {
+            ImGui::Text("List Modules \nSelect your modules desired.");
+            static const char *list_modules[] = {
```

Comment:
```suggestion
            static const char *MODULE_LIST[] = {
```

Reply:
Why in shift ? list_sys_lang is not shift

Reply:
const things should always be capitalized

Reply:
all list inside imgui demo don't have capitalized, and list_sys_lang is like this also, and whith my search on web for code some thing, i have see very lot list like this, and never see shifted

Reply:
We are unique. Also this is static const, like constexpr. If there are places where it's static const but isnt capitalized, we can change it